types/twine-sugarcube/macro.d.ts

```diff
@@ -119,8 +142,20 @@ export interface MacroContext {
 
 export interface MacroDefinition {
     handler: (this: MacroContext) => void;
+    /**
+     * Having this property signifies that this is a container macro
+     * This should be an array of child tag names or `null`
+     * @since 2.0.0
+     */
     tags?: string[] | null | undefined;
```

Comment:
Just looking at this (not a MinusGix's change), however, do we really need an explicit `undefined` here?

1. The type is already marked as optional, so can be implicitly `undefined`.
2. Accepting a value of `undefined`, while legal because JavaScript, isn't really how I'd define its signature.

The following seems like a better fit to me:

```typescript
tags?: string[] | null;
```